src/Interfaces/IGrazing.cs

```diff
@@ -5,6 +5,7 @@ public interface IGrazing
     {
         //A double to store how much grass a grazing animal can eat per day
         double GrassPerDay { get; set; }
+        string Type{get;}
```

Comment:
I think we already have `Type` property on our `IResource` interface-- why do we need both?